pacta/cj_build_tag.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+clones="PACTA_analysis create_interactive_report StressTestingModelDev pacta-data"
+url="git@github.com:2DegreesInvesting/"
+tag="${1:-latest}"
+
+for repo in ${clones}
+do
+    remote="${url}${repo}.git"
+    git clone -b master ${remote} --depth 1
+    echo "--"
+done
+
+for repo in ${clones}
+do
+    echo "${repo} HEAD sha:"
+    git -C "${repo}" rev-parse HEAD
+    echo "--"
+done
+
+for repo in ${clones}
+do
+    git -C "${repo}" tag -a "${tag}" -m "Release pacta ${tag}" HEAD
+    echo "${repo}"
+    echo "$(git -C ${repo} log --pretty='%h %d <%an> (%cr)' | head -n 1)"
+    echo "--"
+done
```

Comment:
This now tags the temporary clone of each repo using the same tag as the container, then prints  a one-line log showing the newly added tag, the commit sha, message, commiter, and time. If the image makes it to production, we can push the tags from inside the final container with `git push origin <tagname>` -- which creates a GitHub release.

```bash
PACTA_analysis
15f92ff  (grafted, HEAD -> master, tag: 0.0.1.9000, origin/master, origin/HEAD) <Clare2D> (5 hours ago)
--
create_interactive_report
6234092  (grafted, HEAD -> master, tag: 0.0.1.9000, origin/master, origin/HEAD) <Clare2D> (84 minutes ago)
--
StressTestingModelDev
f89886d  (grafted, HEAD -> master, tag: 0.0.1.9000, tag: 0.0.1, origin/master, origin/HEAD) <jacobvjk> (6 days ago)
--
pacta-data
5e8ed2e  (grafted, HEAD -> master, tag: 0.0.1.9000, tag: 0.0.1, origin/master, origin/HEAD) <Clare2D> (20 hours ago)

```

Reply:
brilliant! thanks